<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDO</class>
 <widget class="QDialog" name="CDO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>427</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Хімічно-небезпечний об'єкт</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>373</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QPushButton" name="random_button">
      <property name="text">
       <string>Згенерувати випадкові дані</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDialogButtonBox" name="dialog_button_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="type_label">
          <property name="text">
           <string>Тип НХР</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="type_combo_box"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="count_label">
          <property name="text">
           <string>Кількість НХР(т)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="count_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>22</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="locality_label">
          <property name="text">
           <string>Характер місцевості</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="locality_combo_box"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="length_label">
          <property name="text">
           <string>Довжина забудови(км)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="length_spin_box">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="shelter_label">
          <property name="text">
           <string>Вид сховища</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="shelter_combo_box"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="height_label">
          <property name="text">
           <string>Висота обвалування</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="height_spin_box">
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="time_label">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>Час аварії</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="time_edit">
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="time">
           <time>
            <hour>4</hour>
            <minute>15</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Координати аварії</string>
      </property>
      <widget class="QLabel" name="x_label">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>30</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="y_label">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>30</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="x_spin_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9000000</number>
       </property>
       <property name="value">
        <number>583</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="y_spin_box">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>55</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9000000</number>
       </property>
       <property name="value">
        <number>114</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
